src/components/ui/SuggestItem.jsx

```diff
@@ -48,16 +49,24 @@ const CategoryItem = ({ category }) => {
 };
 
 const PoiItem = ({ poi }) => {
-  const { name, className, subClassName, type, alternativeName } = poi;
+  const { name, className, subClassName, type } = poi;
   const icon = IconManager.get({ className, subClassName, type });
+  const [streetAddress, setStreetAddress] = React.useState(null);
+
+  useLayoutEffect(() => {
+    (async () => {
+      const streetAddress = await getStreetAddress(poi);
```

Comment:
This function may trigger a request to the server for items with no `address` object, such as POIs stored in local storage with an older format. In that case, a request is triggered for each item and each keystroke to fetch the address (actually twice per keystroke, as the render seems to be executed twice).
This also causes flickering of the list.

Maybe we would need a mechanism (in `PoiStore` ?) to update the local storage with the new format when a POI is stored with an obsolete format.

Reply:
We now display `alternativeName` when it exists. Is it okay for you ?

Reply:
Yes fine :+1: 
Migrrating the data present in local storage is definitely out of the scope of this PR.